Source/Documentation/Manual/driving.rst

```diff
@@ -763,6 +763,45 @@ if the trains are far away, can lead to memory overflows.
 If after a train selection you click on ``Follow`` the dispatcher window 
 will remain centered on that train.
 
+.. _driving-dispatcher-for-ai-trains:
+
+Using dispatcher window for AI trains
+-------------------------------------
+
+What is described here is valid only for activity mode and explore in 
+activity mode.
+
+There are cases where it would be advisable to re-route an AI train to manage standoffs, 
+train passings, train priorities.
+In this case, using the dispatcher window it is possible to re-route an AI train (e.g. on a siding) 
+and then to get it back on the original route. The feature anyhow also allows to 
+re-route it without getting it back on the original route.
+
+It is suggested to look at this video which explains some practical case 
+https://youtu.be/-f0XVg7bSgU before continuing reading.
+
+To perform this correctly and in a way more similar to reality, some rules have to 
+be followed.
+The concept is that switches must be manually thrown only if they aren't reserved by a train. 
+To be sure of this it is necessary to force to stop the las signal(s) between train(s) 
```

Comment:
```suggestion
To be sure of this it is necessary to force to stop the **las** signal(s) between train(s) 
```
is this a typo?

Reply:
Yes, it is. It must be corrected to "last" .

Reply:
I'll prepare a further commit where I also add something else